pkg/iut/v1alpha1/v1alpha1.go

```diff
@@ -207,17 +209,16 @@ func (h V1Alpha1Handler) Stop(w http.ResponseWriter, r *http.Request, ps httprou
 	logger := h.logger.WithField("identifier", identifier).WithContext(r.Context())
 
 	var stopReq StopRequest
-	defer r.Body.Close()
```

Comment:
Body needs to be closed. Should also be moved down below the `json.NewDecode(r.Body)....` code (after line 217)